core/src/main/java/org/elasticsearch/snapshots/SnapshotInfo.java

```diff
@@ -43,7 +45,7 @@
 /**
  * Information about a snapshot
  */
-public final class SnapshotInfo implements Comparable<SnapshotInfo>, ToXContent, Writeable {
+public final class SnapshotInfo implements Comparable<SnapshotInfo>, ToXContentObject, Writeable {
```

Comment:
hmmm good point, I'll revert this back to `ToXContent` for now. There are other classes that I've skipped for this same reason and will come back to once the easy ones are done